### DISCUSSION TITLE
Implement options for easier styling of breadcrumbs

### DISCUSSION THREAD

**Discussion by rbw14** [felix-berlin/astro-breadcrumbs#177] [Ideas]
Hi Felix,
first of all thanks so much for providing this great repo and investing your time! :) I am really happy I found it!

I wanted to pitch the idea for allowing easier(?) styling of the different breadcrumbs as single elements, e.g. with Tailwind or plain CSS. Here's an example:

![screenshot_breadcrumbs_styling](https://github.com/felix-berlin/astro-breadcrumbs/assets/148999756/f63ca7c4-9a63-4ac4-be67-322f33e264e1)

Maybe this is already possible anyway and I did not see it (I am quite new in the dev sector), but also didn't find anything on this in the documentation. - If I missed anything, any help is greatly appreciated. Otherwise, maybe it is an idea for one of the next updates..? :)

Cheers & please keep up the great work,
Markus

**Comment by felix-berlin** [comment]
Hey @rbw14 ,
thanks for your kind words and suggestions.

You are right, the attributes of the individual breadcrumb items are currently not customizable in "Zero Config Mode". I assume that you want to assign typical Tailwind utility classes.

If you build your breadcrumbs manually using the crumbs property, you have the option of assigning additional attributes there. See: https://docs.astro-breadcrumbs.kasimir.dev/configuration/properties/#crumbs

**Comment by felix-berlin** [comment]
@rbw14 Since I haven't worked much with Tailwind so far, I don't know all the possibilities, which makes it difficult for me to plan this integration.

Ideally, there will be someone else who can bring this feature into the project.

Until then, I think you should be able to implement your style customizations quite well via Vanilla CSS or SCSS.
